CMakeLists.txt

```diff
@@ -1,33 +1,20 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.1.3)
 project(urg_stamped)
 
-## Find catkin and any catkin packages
-find_package(catkin REQUIRED
-  COMPONENTS
-    roscpp
-
-    sensor_msgs
+set(CATKIN_DEPENDS
+  roscpp
+  sensor_msgs
 )
+
+## Find catkin and any catkin packages
+find_package(catkin REQUIRED COMPONENTS ${CATKIN_DEPENDS})
 find_package(Boost 1.53 REQUIRED system chrono thread)
 
 catkin_package(
-  CATKIN_DEPENDS
-    roscpp
-
-    sensor_msgs
+  CATKIN_DEPENDS ${CATKIN_DEPENDS}
 )
 
-include(CheckCXXCompilerFlag)
```

Comment:
Can we drop C++0x support in other repositories?

Reply:
`set(CMAKE_CXX_STANDARD 11)` is used instead.
(it was added in cmake 3.1 and should be better for latest cmake.)
But, I found that we can't upgrade `cmake_minimum_required` to 3.1 due to Debian Jessie.

- Ubuntu Trusty and ROS Indigo: cmake 2.8.12, gcc 4.6.4 (EOL)
- Debian Jessie and ROS Kinetic: cmake 3.0.2, gcc 4.9.2 (~June, 2020)
- Ubuntu Xenial and ROS Kinetic: cmake 3.5.1, gcc 5.3.1 (~April, 2021)
- Debian Stretch and ROS Melodic: cmake 3.7.2, gcc 6.3.0 (~June, 2022)
- Ubuntu Bionic and ROS Melodic: cmake 3.10.2, gcc 7.3.0 (~May, 2023)

`-std=c++0x` was for `GCC<4.8`; this can be dropped, anyway.
I'll downgrade `cmake_minimum_required` to 3.0 and use `add_compile_options(-std=c++11)`.
(`add_compile_options` was added in cmake 3.0)

Reply:
I see. Thank you.